Orange/widgets/data/owselectrows.py

```diff
@@ -68,7 +67,7 @@ def decode_setting(self, setting, value, domain=None):
                 op, values = condition[-2:]
 
                 var = attr in domain and domain[attr]
-                if var and var.is_continuous and not isinstance(var, TimeVariable):
+                if var and var.is_continuous:
```

Comment:
Be careful when solving the conflict with #4798: there's no `var` but just `attr`, and it can be a `str` or `Variable`, so you probably want something like

```python
if isinstance(attr, ContinuousVariable) \
    or isinstance(attr, str) and OWSelectRows.AllTypes.get(attr) == CONTINUOUS
```